case/infrasim/virtual_node/T33250_idic_KCSFruSensorList.py

```diff
@@ -0,0 +1,64 @@
+from case.CBaseCase import *
+from lib.SSH import CSSH
+from lib.Apps import dhcp_query_ip
+import re
+
+class T33250_idic_KCSFruSensorList(CBaseCase):
+    '''
+    [Purpose ]: Boot OS in InfraSIM host and send ipmi command.
+    [Author  ]: june.zhou@emc.com
+    '''
+    def __init__(self):
+        CBaseCase.__init__(self, self.__class__.__name__)
+
+    def config(self):
+        CBaseCase.config(self)
+
+        # Config all node to go to host.
+        self.enable_hypervisor_ssh()
+
+    def test(self):
+        # Because disk vmdk file will be locked by hypervisor, this
+        # case only take one node and test.
+        self.test_node = self.stack.random_node()
+
+        # Mount disk vmdk to virtual node
+        # Boot all nodes QEMU host to disk
+        try:
+            self.stack.boot_to_disk(self.test_node, disk_image=self.data['disk_image'])
+        except Exception, e:
+            self.result(BLOCK, 'Fail to boot node to disk: {}'.format(e))
+            return
+        try:
+            qemu_conn = self.stack.get_host_ssh(self.test_node,
+                                                username=self.data['host_username'],
+                                                password=self.data['host_password'],
+                                                dhcp_server=self.data['dhcp_server'],
+                                                dhcp_user=self.data['dhcp_username'],
+                                                dhcp_pass=self.data['dhcp_password'])
+        except Exception, e:
+            self.result(BLOCK, 'Fail to get host access: {}'.format(e))
+            return
+
+        qemu_conn.remote_shell('echo {} | sudo -S modprobe ipmi_devintf'.format(self.data['host_password']))
+        qemu_conn.remote_shell('echo {} | sudo -S modprobe ipmi_si'.format(self.data['host_password']))
+
+        rsp = qemu_conn.remote_shell('echo {} | sudo -S ipmitool sensor list'.format(self.data['host_password']))
+
+        self.log('INFO', 'rsp: \n{}'.format(rsp))
+
+        #To match "System Fan" and "PS Status",
+        # as each system will have Fan and PS sensors in sensor list if sensor list is correct.
+        is_match_fan = re.search(r'System Fan', format(rsp))

```

Comment:
Hi, @JuneZhou19 , as I know, not all product name system fan as "System Fan", there may be "SYS_Fan0", xxx, xxx.
You could expect other key information in this test.